File: App-Revistas/src/app/Objetos/Pago.ts
export class Pago {

    idPago: number;
    parte_obtenido: number;
    parte_editor: number;
    total: number;

    constructor(
        idPago: number,
        parte_obtenido: number,
        parte_editor: number,
        total: number
    ) {
        this.idPago = idPago;
        this.parte_obtenido = parte_obtenido;
        this.parte_editor = parte_editor;
        this.total = total;
    }
}

File: BackendRevistas/src/main/java/EntidadesAuxiliares/Pago.java
package EntidadesAuxiliares;

/**
 *
 * @author marco
 */
public class Pago {

    private int idPago;
    private Double parte_obtenido;
    private Double parte_editor;
    private Double total;

    public Pago(int idPago, Double parte_obtenido, Double parte_editor, Double total) {
        this.idPago = idPago;
        this.parte_obtenido = parte_obtenido;
        this.parte_editor = parte_editor;
        this.total = total;
    }

    public int getIdPago() {
        return idPago;
    }

    public void setIdPago(int idPago) {
        this.idPago = idPago;
    }

    public Double getParte_obtenido() {
        return parte_obtenido;
    }

    public void setParte_obtenido(Double parte_obtenido) {
        this.parte_obtenido = parte_obtenido;
    }

    public Double getParte_editor() {
        return parte_editor;
    }

    public void setParte_editor(Double parte_editor) {
        this.parte_editor = parte_editor;
    }

    public Double getTotal() {
        return total;
    }

    public void setTotal(Double total) {
        this.total = total;
    }
}
